babybuddy/middleware.py

```diff
@@ -38,6 +39,34 @@ def update_en_us_date_formats():
         custom_input_formats + formats_en_us.DATETIME_INPUT_FORMATS
 
 
+def update_en_gb_date_formats():
+    if settings.USE_24_HOUR_TIME_FORMAT:
+        # 25 October 2006 14:30:00
+        formats_en_gb.DATETIME_FORMAT = 'j F Y H:i:s'
+        custom_input_formats = [
+            '%d/%m/%Y %H:%M:%S',  # '25/10/2006 14:30:59'
+            '%d/%m/%Y %H:%M',  # '25/10/2006 14:30'
+        ]
+        formats_en_gb.SHORT_DATETIME_FORMAT = 'd/m/Y H:i'
+        formats_en_gb.TIME_FORMAT = 'H:i'
+    else:
+        formats_en_gb.DATETIME_FORMAT = 'j F Y f a'  # 25 October 2006 2:30 p.m
+        # These formats are added to support the locale style of Baby Buddy's
+        # frontend library, which uses momentjs.
+        custom_input_formats = [
+            '%d/%m/%Y %I:%M:%S %p',  # '25/10/2006 2:30:59 PM'
+            '%d/%m/%Y %I:%M %p',  # '25/10/2006 2:30 PM'
+        ]
+
+    # Not setting this makes pages display using the the 25/10/2006 as desired
+    # Add custom "short" version of `MONTH_DAY_FORMAT`.
+    # formats_en_gb.SHORT_MONTH_DAY_FORMAT = 'j M'
```

Comment:
@Amith211 this is optional. I do it specifically got en(-US) because the normal `MONTH_DAY` format uses full month names. If en-GB doesn't need the same treatment you can go ahead and remove this (and the relevant test).